### DISCUSSION TITLE
Error on jinja render for matched documents created by python-logstash-async

### DISCUSSION THREAD

**Discussion by drakaru** [jertel/elastalert2#310] [Q&A] [answered]
We use python-logstash-async in our python services, which adds a 'type' field to the document 
https://github.com/eht16/python-logstash-async/blob/master/logstash_async/formatter.py#L104

Since #282 both the match and the rule are expanded into the same function call. 
They clash since they both have a field called type

> TypeError: render() got multiple values for keyword argument 'type'

**Comment by mrfroggg** [comment, marked as answer]
If I check on how it works with `alert_text_args` works in `BasicMatchString`, it prioritizes the content of the match before the content of the rule.
We'll have to do something similar for `alert_text_jinja`.

**Reply by mrfroggg** [reply]
Fixed in #311
